public/version-aliases/go.json

```diff
@@ -14,10 +14,10 @@
   "1.6.x": "1.6.4",
   "1.7.6": "1.7.6",
   "1.7.x": "1.7.6",
-  "1.8.3": "1.8.3",
-  "1.8.x": "1.8.3",
-  "1.9": "1.9",
```

Comment:
Is `1.9` removed intentionally?

Reply:
I'm not sure why there was a `1.9` :arrow_right: `1.9` "alias" in there before.  If someone specifies a literal `1.9`, then no alias will match, meaning they will get `1.9`.